docker_services_cli/env.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Docker-Services-CLI is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Environment module."""
+
+import os
+import sys
+from distutils.version import StrictVersion
+
+import click
+
+
+def _setdefault_env(variable, default_value):
+    """Set environmental variable value if it does not exist."""
+    os.environ[variable] = os.environ.get(variable, default_value)
+
+
+def _is_version(version):
+    """Checks if a string is a version of the format `x.y.z`.
+
+    NOTE: It is not mandatory to be up to patch level. The following would be accepted:
+    - 10.1
+    - 9
+    - 15.0.1a2
+    """
+    try:
+        # StrictVersion fails on plain numbers (e.g. "10")
+        if version.isnumeric():
+            return True
+        StrictVersion(version)
+        return True
+    except ValueError:
+        return False
+
+
+def _setversion_env(variable, default_value):
+    """Set a specific service version from the environment.
+
+    It parses the value to distinguish between a version and a defined latest.
+    NOTE: It requires that all variables for latest versions have been set up.
+    """
+    version_value = os.environ.get(variable, default_value)
+    if _is_version(version_value):
+        os.environ[variable] = version_value
+    else:
+        version_name = version_value
+        version_value = os.environ.get(version_name)
+        if not version_value:
+            click.secho(
+                f"Environment variable for version {version_name} not set.", fg="red"
+            )
+            sys.exit(-1)
+        else:
+            os.environ[variable] = version_value
+
+
+def export_env_vars():
+    """Export the environment variables for services and versions."""
+    # export variables for latest versions
+    _setdefault_env("ES_5_LATEST", "5.6.10")
```

Comment:
I would create a `config.py` where we would have a mapping for this defaults e.g.:
```python
version_default = {
   "ES_6_LATEST", "6.8.12",
   "ES_7_LATEST": "7.9.0",
   ....
}
```

- This code would be simpler.
- It would be more UX friendly to go and change `config.py` for version defaults.